### PR TITLE
Default to using Solr real-time get for retrieving single documents

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: spec/controllers/catalog_controller_spec.rb [hunk 1]
```diff
@@ -251,7 +251,7 @@
         get :show, params: { id: doc_id, format: 'json' }
         expect(response).to be_success
         json = JSON.parse response.body
-        expect(json["response"]["document"].keys).to match_array(["author_t", "opensearch_display", "marc_display", "published_display", "author_display", "lc_callnum_display", "title_t", "pub_date", "pub_date_sort", "subtitle_display", "format", "url_suppl_display", "material_type_display", "title_display", "subject_addl_t", "subject_t", "isbn_t", "id", "title_addl_t", "subject_geo_facet", "subject_topic_facet", "author_addl_t", "language_facet", "subtitle_t", "timestamp"])
+        expect(json["response"]["document"].keys).to match_array(["author_t", "marc_display", "published_display", "author_display", "lc_callnum_display", "title_t", "pub_date", "subtitle_display", "format", "url_suppl_display", "material_type_display", "title_display", "subject_addl_t", "subject_t", "isbn_t", "id", "title_addl_t", "subject_geo_facet", "subject_topic_facet", "author_addl_t", "language_facet", "subtitle_t", "timestamp"])
```
Comment (cbeer) [inline spec/controllers/catalog_controller_spec.rb]: Apparently Solr doesn't return any copy fields in the RTG response.

Comment (jkeck) [inline spec/controllers/catalog_controller_spec.rb]: This should be okay though right?  I imagine a copy field would be an odd choice for displaying on the record view.

Comment (cbeer) [inline spec/controllers/catalog_controller_spec.rb]: Yeah, agreed (and apparently so do the Solr committers), although @erikhatcher has reported it upstream already: https://issues.apache.org/jira/browse/SOLR-8767. If someone does need that behavior, all they need to do is change their defaults.

Comment (mejackreed) [inline spec/controllers/catalog_controller_spec.rb]: Agree here with @jkeck but feels a little bad for some reason. Whats the reason to do the real-time?

Comment (cbeer) [inline spec/controllers/catalog_controller_spec.rb]: It's there by default, instead of our kinda clunky custom request handler.

**Pull request merged**